<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DropboxSettingsPage</class>
 <widget class="QWidget" name="DropboxSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropbox</string>
  </property>
  <layout class="QVBoxLayout" name="layout_dropboxsettingspage">
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>Strawberry can play music that you have uploaded to Dropbox</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enable">
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LoginStateWidget" name="login_state" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_authorization" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>28</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_login">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_login">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_buttons">
     <item>
      <widget class="QPushButton" name="button_reset">
       <property name="text">
        <string>Reset cursor and songs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>357</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
